specification/cdn/resource-manager/Microsoft.Cdn/preview/2022-05-01-preview/cdn.json

```diff
@@ -477,6 +477,14 @@
           }
         ],
         "responses": {
+          "200": {
+            "description": "Ok and the operation will complete synchronously.",
+            "headers": {
+              "location": {
```

Comment:
Why would 200 need location header? Normally only 202 does. Also, could you add '200' response in the example ./examples/Profiles_MigrationCommit.json